packages/design-system/src/Avatar/Avatar.type.ts

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from "react";
+
+export type Variant = "circular" | "rounded" | "square" | "string";
+export type Size = "small" | "medium" | "large" | "string";
+
+export type AvatarProps = {
+  variant?: Variant;
+  src: string;
+  imgProps?: object;
```

Comment:
`imgProps` doit avoir les `props ou attributs` natifs d'un element `img` je pense. L'interface doit etre [HTMLImageElement](https://reference.codeproject.com/dom/htmlimageelement)

Reply:
On peut aussi enlever les props comme `src`, `alt`. Qu'es-ce que t'en penses ?

Reply:
Pv pour une discution.

Reply:
on met quoi si on enlève les src et alt?

Reply:
Je parle d'un truc du genre 
```suggestion
  imgProps?: Omit<HTMLImageElement, 'src' | 'alt'>;
```